.github/workflows/crowdin-to-github.yml

```diff
@@ -0,0 +1,31 @@
+name:  Crowdin To GitHub
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '15 * * * *' # sync every hour
```

Comment:
i think it's enough to run this once a day, as the changes aren't released that fast anyway. Or would there be a reason to do this every hour?